scripts/jenkins-yoctobuild-build.sh

```diff
@@ -148,13 +148,14 @@ build_custom_qemu() {
     #   https://bugs.launchpad.net/qemu/+bug/1481272
     # installing from source doesn't exhibit this behaviour
 
-    if [ -f /var/tmp/qemu-built ]; then
-        echo "qmeu already built"
-        return
-    fi
-
-    git clone git://git.qemu.org/qemu.git
-    cd qemu && git checkout b187e2b53055007fa08ccb9fb120578bae0d02f3
+    #if [ -f /var/tmp/qemu-built ]; then
+    #    echo "qmeu already built"
+    #    return
+    #fi
```

Comment:
I don't know if the droplets have some sort of data persistence. Also, figured it's safer to build qemu every time rather than end up with a previous build which may have been done using another qemu revision.